mflowgen/common/rtl/gen_rtl.sh

```diff
@@ -56,32 +56,39 @@ else
            cat genesis_verif/* >> design.v
          else
            cp garnet.v design.v
-         fi"
+         fi
+         cat global_buffer/rtl/*.sv >> design.v
+         cat global_buffer/rtl/*.svh >> design.v"
       # Copy the concatenated design.v output out of the container
       docker cp $container_name:/aha/garnet/design.v ../outputs/design.v
       # Kill the container
       docker kill $container_name
       echo "killed docker container $container_name"
       cd ..
-    
-    # Else we want to use local python env to generate rtl 
+
+    # Else we want to use local python env to generate rtl
     else
       current_dir=$(pwd)
       cd $GARNET_HOME
-       
+
       eval "python garnet.py $flags"
-      
+
       # If there are any genesis files, we need to cat those
       # with the magma generated garnet.v
       if [ -d "genesis_verif" ]; then
         cp garnet.v genesis_verif/garnet.sv
         cat genesis_verif/* >> $current_dir/outputs/design.v
+
       # Otherwise, garnet.v contains all rtl
       else
         cp garnet.v $current_dir/outputs/design.v
       fi
-    fi 
-    
+
+      # Copy global buffer systemverilog from the global buffer folder
+      cat global_buffer/rtl/*.sv >> $current_dir/outputs/design.v
+      cat global_buffer/rtl/*.svh >> $current_dir/outputs/design.v
```

Comment:
Can we make this copy the global_controller rtl as well?

Reply:
It should be already included as it copies all files in `genesis_verif` folder.